CHANGES.markdown

```diff
@@ -1,3 +1,7 @@
+## Changes in 0.7.0
+ - Add functions and orphan instances introduced by changes to
+   `base-4.7.0.0` and `base-4.8.0.0`
```

Comment:
Let's already update the changelog for the next release.  I tend to put it under a header

```makdown
## next
 - Add `Alt` data type to `Data.Monoid.Compat`
...
```